frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -131,6 +144,18 @@ export function ManagePublicKey() {
         (sender && !fileSubmitted && !hasPublicKey) || uploadNewPublicKey;
     const hasUploadError = fileSubmitted && !isUploading && !isSuccess;
 
+    if (isSuccess) {
+        trackAppInsightEvent(featureEvent, {
+            fileUpload: {
+                status: "Success",
```

Comment:
Should we make these statuses some sort of `enum`?

Reply:
I don't think its necessary for one var.